src/components/NamespaceFilter/NamespaceFilter.tsx

```diff
@@ -49,14 +41,25 @@ export class ServiceFilter extends React.Component<ServiceFilterProps, ServiceFi
       filterValues: []
     };
 
+    let initialFilters = this.initialFilterList(namespaceFilter);
+
     this.state = {
-      currentFilterType: serviceNameFilter,
-      filterTypeList: [serviceNameFilter, namespaceFilter],
-      activeFilters: ServiceFilterSelected.getSelected(),
+      currentFilterType: initialFilters[0],
+      filterTypeList: initialFilters,
+      activeFilters: NamespaceFilterSelected.getSelected(),
       currentValue: ''
     };
   }
 
+  initialFilterList(namespaceFilter: FilterType) {
```

Comment:
nitpicking, but good to know, you can copy an array more easily with `splice`:
`return this.props.initialFilters.splice().concat(namespaceFilter)`

Reply:
sorry, not splice, slice. I always mix them up :/
`return this.props.initialFilters.slice().concat(namespaceFilter)`